website/static/js/fangorn.js

```diff
@@ -12,6 +12,8 @@ var tbOptions;
 
 var tempCounter = 1;
 
+var uploadInProgress = false;
```

Comment:
It's not a global in that sense because it's not exposed to window, it's a global within fangorn because commonJS compartmentalizes this.  But I see your point, it would be a problem if we had multiple instances of Fangorn in a single page.I'll see what I can do, a lot of these files are called with .call and their this is defined already. Any suggestions welcome.